README.md

```diff
@@ -135,7 +135,7 @@ The `merchantKey` can be any DICT key:
 
 - **CPF**: 12345678900 (11 characters, no separators)
 
-- **CPF**: 00038166000105 (14 characters, no separators)
+- **CNPJ**: 00038166000105 (14 characters, no separators)
```

Comment:
Here, was typed `**CPF**`, when you probably meant `**CNPJ**`  :)

Reply:
Yep, my bad, thanks for noticing!